editor/src/components/inspector/sections/layout-section/self-layout-subsection/frame-updating-layout-section.spec.browser2.tsx

```diff
@@ -0,0 +1,1029 @@
+import {
+  BakedInStoryboardUID,
+  BakedInStoryboardVariableName,
+} from '../../../../../core/model/scene-utils'
+import type {
+  LocalRectangle,
+  MaybeInfinityLocalRectangle,
+  MaybeInfinityRectangle,
+} from '../../../../../core/shared/math-utils'
+import { canvasRectangle, localRectangle } from '../../../../../core/shared/math-utils'
+import {
+  filtered,
+  fromField,
+  traverseArray,
+} from '../../../../../core/shared/optics/optic-creators'
+import { forEachOf, toArrayOf } from '../../../../../core/shared/optics/optic-utilities'
+import type { EditorRenderResult } from '../../../../canvas/ui-jsx.test-utils'
+import {
+  TestAppUID,
+  TestSceneUID,
+  formatTestProjectCode,
+  getPrintedUiJsCode,
+  getPrintedUiJsCodeWithoutUIDs,
+  renderTestEditorWithCode,
+} from '../../../../canvas/ui-jsx.test-utils'
+import * as EP from '../../../../../core/shared/element-path'
+import { selectComponentsForTest } from '../../../../../utils/utils.test-utils'
+import { RegisteredCanvasStrategies } from '../../../../canvas/canvas-strategies/canvas-strategies'
+import { act, fireEvent } from '@testing-library/react'
+
+function downClickUp(targetElement: HTMLElement, mouseEventInit: MouseEventInit) {
+  fireEvent(targetElement, new MouseEvent('mousedown', mouseEventInit))
+  fireEvent(targetElement, new MouseEvent('click', mouseEventInit))
+  fireEvent(targetElement, new MouseEvent('mouseup', mouseEventInit))
+}
```

Comment:
Please use [`mouseClickAtPoint`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/event-helpers.test-utils.tsx#L347) for this

Reply:
Fixed in 01dd60249.